driver/src/main/com/mongodb/ClientSession.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2017 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.mongodb;
+
+import com.mongodb.annotations.NotThreadSafe;
+import org.bson.BsonDocument;
+import org.bson.BsonTimestamp;
+
+import java.io.Closeable;
+
+/**
+ * A client session.
+ *
```

Comment:
We are going to need to link this to some documentation in the future.

Add: 

```
 * @mongodb.server.release 3.6
 * @since 3.6
```

Reply:
Done